bokeh/core/_templates/autoload_tag.html

```diff
@@ -21,9 +21,4 @@
     This script injects a ``<div>`` in place, so must be placed under ``<body>``.
 
 #}
-<script
-    src="{{ src_path }}"
-    id="{{ elementid }}"
-    data-bokeh-model-id="{{ modelid }}"
```

Comment:
This breaks the API specification [server_session](http://bokeh.pydata.org/en/latest/docs/reference/embed.html#bokeh.embed.server_session), breaks my code that spread a few figures/tables to respective divs inside my html template. Now every such div contains all figures from the document!

Reply:
@complyue please join the discussion at https://github.com/bokeh/bokeh/issues/8499 to have a voice in getting this kind of functionality added back in a supported, maintainable way.